### ISSUE TITLE
[BUG] Sorted sets with long names crash KeyDB when using ZRANGESTORE

### ISSUE THREAD

**Issue by Firaenix** [EQ-Alpha/KeyDB#393]
**Describe the bug**

Title describes the bug.

**To reproduce**

```
ZADD longnameset 10 "sha256-hash:2a553c8a93af1a1043aeb624de5e484af683d493e2e035343e7bd732511deeee" 20 "world"

ZRANGE longnameset 10 +inf BYSCORE WITHSCORES
1) "sha256-hash:2a553c8a93af1a1043aeb624de5e484af683d493e2e035343e7bd732511deeee"
2) "10"
3) "world"
4) "20"

ZRANGESTORE longsubset longnameset 10 +inf BYSCORE
Error: Server closed the connection
```

**Expected behavior**

A new sorted set should be created  with the results of the ZRANGE command.

**Additional information**

Can reproduce on the following docker containers:
- eqalpha/keydb:arm64_v6.2.2
- eqalpha/keydb:x86_64_v6.2.2

**Comment by Firaenix** [comment]
Upon further looking, the max character length for a ZSET entry is 64 characters long

![image](https://user-images.githubusercontent.com/8898038/152085098-5c41b1d8-5516-4d0c-a1d3-b6086193aad8.png)

**Comment by Firaenix** [comment]
![image](https://user-images.githubusercontent.com/8898038/152085841-b1ad9e80-e2d2-4f8a-868b-003f7aa636d1.png)

**Comment by VivekSainiEQ** [comment]
Looks like this issue has been resolved with #408, closing this thread.
